packages/debug-ui/scripts/deploy-newsroom.ts

```diff
@@ -1,22 +1,13 @@
 import { Civil } from "@joincivil/core";
-import * as fs from "fs";
-
-(async () => {
-    const article = fs
-        .readFileSync("assets/article.md")
-        .toString();
+import { EthAddress } from "../../core/build/src/types";
```

Comment:
do an `export * from "./types"` in index.ts in core and use `import { EthAddress } from "@joincivil/core"` here